multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -389,14 +405,71 @@ const advanceAndSettleScenario = test.macro({
       ),
     );
 
-    await assertTxStatus('DISBURSED');
+    await assertTxStatus(evidence.txHash, 'DISBURSED');
   },
 });
 
 test.serial(advanceAndSettleScenario, LP_DEPOSIT_AMOUNT / 4n, 'osmosis');
 test.serial(advanceAndSettleScenario, LP_DEPOSIT_AMOUNT / 8n, 'noble');
 test.serial(advanceAndSettleScenario, LP_DEPOSIT_AMOUNT / 5n, 'agoric');
 
+test.skip('advance failed (e.g. to missing chain)', async t => {
+  const mintAmt = LP_DEPOSIT_AMOUNT / 10n;
+  const {
+    assertTxStatus,
+    makeFakeEvidence,
+    nobleTools,
+    nobleAgoricChannelId,
+    txOracles,
+    vstorageClient,
+  } = t.context;
+
+  // EUD wallet on the specified chain
+  const eudWallet = await createWallet('cosmos1');
```

Comment:
```suggestion
  const eudWallet = await createWallet('cosmos');
```
Nit: this doesn't seem to affect test fidelity, but the argument for `createWallet` is just the bech32 prefix.

Reply:
Fixed